backend/doc/setting.md

```diff
@@ -471,10 +471,25 @@ With a Collabora host, `<collabora_host>` may look like `collaboradomain.ndd` or
 
 To enable online edition on Tracim and allow communication with your edition software.
 
-First you need to enable the edition on the API:
+First you need to enable the edition on the API. To do that you have to add the application to app_enabled. For example:
 
-    collaborative_document_edition.activated = True
-    collaborative_document_edition.software = collabora`
+    Before:
+    app.enabled = contents/thread,contents/html-document,contents/folder,agenda
+
+    After:
+    app.enabled = contents/thread,contents/html-document,contents/folder,agenda,collaborative_document_edition
+
+This application need contents/file application to work fully. You will have to add this application too. With our example:
+
+    Before:
+    app.enabled = contents/thread,contents/html-document,contents/folder,agenda,collaborative_document_edition
+
+    After:
+    app.enabled = contents/thread,contents/file,contents/html-document,contents/folder,agenda,collaborative_document_edition
+
+Once the application enabled, you will have to specify which software you desire to use. So far Tracim only support Collabora.
+
+    collaborative_document_edition.software = collabora
```

Comment:
to make sure : this is not directly related to the call feature, right.

Reply:
Yes, it's a fix of an error that I just spotted. I didn't get an answer on [this comment](https://github.com/tracim/tracim/pull/5089#discussion_r749285244), but I had a discussion with @inkhey who had no real solution about "how explain how to add an application in the doc".

Reply:
ok :)